app/actions.ts

```diff
@@ -0,0 +1,18 @@
+'use server';
+import { cookies } from 'next/headers';
+import { deleteSessionByToken } from '../database/sessions';
+
+export async function logout() {
+  const cookieStore = cookies();
+  const token = cookieStore.get('sessionToken');
+
+  if (token) {
+    await deleteSessionByToken(token.value);
+  }
```

Comment:
move `await ...` up to the previous line and remove the `{` and `}`?

Reply:
a710fc8